lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -83,12 +104,28 @@ dampening_suppress_time:
 
 default_information:
   kind: boolean
-  config_get_token_append: '/^default-information originate$/'
-  config_set_append: '<state> default-information originate'
-  default_value: false
+  cli_nexus:
+    config_get_token_append: '/^default-information originate$/'
+    config_set_append: ' <state> default-information originate'
+    default_value: false
+  cli_ios_xr:
+    # IOS XR does not have default-information under address-family,
+    #   so we must replace config get/set
+    # Perhaps better to fix in ruby code?
```

Comment:
Not sure what the right fix is?  Currently there is a check for vrf-unsupported features under XR, so I imagine this could be added as an address-family-unsupported feature?  But the other fix feels like a bit of a hack to begin with.  Any advice?